sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -166,11 +168,14 @@ export class StudentService extends RecordDataModelService<Student> {
     student.sinConsent = studentInfo.sinConsent;
     try {
       // Get PD status from SFAS integration data.
-      student.studentPDVerified = await this.sfasIndividualService.getPDStatus(
-        user.lastName,
-        student.birthDate,
-        studentSIN,
-      );
+      const sfasStudentPDVerified =
+        await this.sfasIndividualService.getPDStatus(
+          user.lastName,
+          student.birthDate,
+          studentSIN,
+        );
+      student.studentPDVerified = sfasStudentPDVerified;
+      student.disabilityStatus = this.getPDStatus(sfasStudentPDVerified);
```

Comment:
Maybe not part of the PR, but, `getPDStatus` will return `undefined` even if the student is not present in SFAS, then does this `try catch` matter?

for that scenario(new student), is the plan to add `not requested` from line 178 and do we need line 177?

Reply:
https://github.com/bcgov/SIMS/pull/2115/files#r1265791873 this can avoid adding status to a new student

Reply:
If the student is not present in SFAS, getPDStatus() (refacored to getDisabilityStatus) will return `Not required` 

![image](https://github.com/bcgov/SIMS/assets/54600590/0c29f91b-6bf8-4bdb-98e8-c2c413b6f503)

The existing try catch I believe is to ensure that, we should not go further to create a student if we cannot get the details from SFAS table as it could end  up creating student with wrong disability status.